packages/kbn-scout/src/playwright/page_objects/maps_page.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { retryForSuccess } from '@kbn/ftr-common-functional-services';
+import { ToolingLog } from '@kbn/tooling-log';
+import { ScoutPage } from '../fixtures/types';
+
+const RENDER_COMPLETE_SELECTOR = '[data-render-complete="true"]';
+const RENDER_COMPLETE_PENDING_SELECTOR = '[data-render-complete="false"]';
+const DATA_LOADING_SELECTOR = '[data-loading]';
+
+const debugLog = ToolingLog.bind(ToolingLog, { level: 'debug', writeTo: process.stdout });
```

Comment:
Scout already provides `log` fixture, I think we need to use that one and it should be available in the same context as `page` fixture.

Reply:
https://github.com/elastic/kibana/pull/204607/commits/d2c8a8a3a86634fbbd6731d431e2a7be91ad39f6